libtransmission/block-info.h

```diff
@@ -14,7 +14,9 @@ struct tr_block_info
 public:
     static auto constexpr BlockSize = uint32_t{ 1024U * 16U };
 
-    tr_block_info() noexcept = default;
+    tr_block_info() noexcept
```

Comment:
Does this actually change anything?

Reply:
Yes, as discussed last week: see the early discussion. The alternative would be to put the private members above that line instead of under that line.